src/routes/userRoutes.test.js

```diff
@@ -68,7 +69,7 @@ describe('Testing user routes', () => {
 
     await resetPassword(req, res)
     expect(res.status.mock.calls[0][0]).toEqual(500)
-    expect(mockJson.mock.calls[0][0]).toEqual({ error: {message: 'Email delivery failed. Please try again'}})
+    expect(res.json.mock.calls[0][0]).toEqual({ error: {message: 'Email delivery failed. Please try again'}})
```

Comment:
same here

Reply:
I like `res.json` better because it looks the same how the tested code does. I think this way is easier for the reviewers tounderstand the code intuitively.

Reply:
yea, makes sense. btw do you expect to test logger info as well?

Reply:
I don't know actually cuz there is only vague idea what to include to the tests. Here is what I think how it is so far.

    1. Write tests for what your tested code is supposed to do
    2. Include tests the way your test covers 100% of tested file
    3. I don't think we should test every single helper function. If so, testing code can be unneccessarily too long and hard to be read

So I didn't expect logger

Reply:
`logger` is best judgement for the developer to put. It doesn't need to be tested. It serves to be informational, not core functionality. Testing should cover core functionality, so logger should be tested only if its part of core functionality. LIke when a database module mess up, that would affect our production systems so maybe a logger there is necessary. There is no 100% rule for this.